.tekton/operator-bundle-pipeline.yaml

```diff
@@ -415,6 +425,8 @@ spec:
       value: "$(params.central-db-image-repo):$(tasks.determine-main-image-tag.results.IMAGE_TAG)"
     taskRef:
       name: wait-for-image
+    # This timeout should be kept in sync with the maximum runtime of the PipelineRun for building the central DB image.
+    timeout: 1h30m
```

Comment:
Values you used are inconsistent between themselves.
For `roxctl` you took `timeouts.pipeline` value for `central-db` you took `timeouts.tasks` value. Much better than me trying to explain, I suggest you check documentation https://tekton.dev/docs/pipelines/pipeline-api/#tekton.dev%2fv1 regarding what each means to make your decision which value to use.

Also, some tasks (e.g. `wait-for-collector-full-image`, `wait-for-scanner-slim-image`) don't have `timeout` attributes. I suppose that's because these pipelines don't exist in the stackrox repo. They will exist after I complete ROX-24468 and there's already a ticket in the epic that depends on it - ROX-26471.
I suggest you add `TODO` comments to each of the `wait-for-*` tasks where the `timeout` isn't defined to not forget to add them.

```
# TODO(ROX-24468): add timeout
# timeout: ?h?m
```

Feel free to use ROX-26471 or create a new ticket.

Reply:
I am now using the pipeline timeout for all the `wait-for-image` tasks.
I have also added the TODO comments
(and updated a related comment).